graders/python/requirements.txt

```diff
@@ -14,14 +14,14 @@ numpy==1.26.4
 openpyxl==3.1.5
 pandas==2.2.3
 Pillow==11.1.0
-PuLP==2.9.0
+PuLP==3.0.2
```

Comment:
The changelog for this looks pretty mild: https://github.com/coin-or/pulp/blob/master/HISTORY